lib/cc/engine/rubocop.rb

```diff
@@ -60,11 +53,13 @@ def local_path(path)
 
       def rubocop_config_store
         @rubocop_config_store ||= begin
-          config_store = RuboCop::ConfigStore.new
-          if (config_file = @engine_config["config"])
-            config_store.options_config = config_file
+          Dir.chdir(@code) do
+            config_store = RuboCop::ConfigStore.new
+            if (config_file = @engine_config["config"])
```

Comment:
Okay. I read this enough times and I understand it. I think we need to figure out what we want before we can decide how to proceed. I'll round up with Francis.